config/rtdb_configuration.xsd

```diff
@@ -105,7 +105,21 @@
             <xs:sequence>
                   <xs:element name="MulticastAddress" type="xs:string"></xs:element>
                   <xs:element name="MulticastPort" type="xs:int"></xs:element>
-                  <xs:element name="Interface" type="xs:string" minOccurs="0"></xs:element>
+                  <xs:element name="Interface" type="xs:string" minOccurs="0" maxOccurs="1"></xs:element>
+                  <xs:element name="InterfacePriorityList" minOccurs="0" maxOccurs="1">
+                        <xs:complexType>
+                              <xs:sequence>
```

Comment:
Use type definition for list. Same type can be used for PriorityList and BlackList.